acceptance/suites/tests/010-puppetserver-cli/subcommand/SERVER-297_common_behavior.rb

```diff
@@ -21,3 +35,13 @@
   assert_match(/\bJARS_REQUIRE\b/, stdout, "JARS_REQUIRE missing")
   assert_match(/\bJARS_NO_REQUIRE\b/, stdout, "JARS_NO_REQUIRE missing")
 end
+
+step "irb: Check that proxy env-variables are present"
+on(master, "echo 'puts JSON.pretty_generate(ENV.to_hash)' | #{proxy_env_vars} puppetserver irb -f -rjson") do
+  assert_match(/\bHTTP_PROXY\b/, stdout, "HTTP_PROXY missing")
```

Comment:
Same comment here on how I think it would be good to assert that the expected value was passed through for each variable.